ui/src/app/applications/components/application-pod-view/pod-view.tsx

```diff
@@ -388,6 +410,29 @@ function formatMetric(name: ResourceName, val: number) {
     return (val || '0') + 'm';
 }
 
+function claculatePodGrouprResquests(pods: Pod[]) {
+    let totalCpuMillicores = 0;
+    let totalMemory = 0;
+
+    pods.forEach(pod => {
+        pod.info?.forEach(info => {
+            if (info.name === 'Requests (CPU)') {
+                const numericValue = parseInt(info.value, 10); //convert the string value to number
+                totalCpuMillicores += numericValue;
+            }
+            if (info.name === 'Requests (MEM)') {
```

Comment:
Let's use constants for these.

Reply:
done

Reply:
This seems like it wasn't committed.

Reply:
🤔 weren't you referring about  the totalvalue variables? or maybe  I misunderstood it.
have changed this to these const

```
let totalCpuMillicores
let totalMemrory
```

```
const resources = {
        cpu: 0,
        memory: 0
    };
```

Reply:
oh ok gottacha, then how about declaring it in utlis as const, and then just importing it?

Reply:
commited it.